.gitignore

```diff
@@ -86,3 +86,12 @@ vendor/assets/javascripts/bootstrap-datepicker.js
 vendor/assets/javascripts/bootstrap-timepicker.min.js
 vendor/assets/javascripts/typeahead.jquery.js
 vendor/assets/javascripts/nprogress.js
+
```

Comment:
I'm wondering if we might need these - the fixtures are a terrible pain for PRs, but I am just finding a situation on another project where is seems like we need them ...

maybe wrong place for detailed discussion of this, but on my mind at the moment (not work for you to do) is that perhaps we need to rename these as "caches" instead of fixtures, and how we can work out a good path for dealing with them ... hmmm ... I'll get into that in a blog post tomorrow ...